http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcProviderMetadata.java

```diff
@@ -330,7 +330,7 @@ public void setClaimTypesSupported(List<String> claimTypesSupported) {
     }
 
     public Boolean getClaimsParameterSupported() {
```

Comment:
Since `OidcProviderMetadata` is currently private API, let's go ahead and change the return value from Boolean to boolean as well.

Reply:
done